node/src/main/scala/coop/rchain/node/configuration/model.scala

```diff
@@ -66,6 +66,7 @@ final case class Deploy(
     privateKey: Option[PrivateKey],
     location: String
 ) extends Command
+final case class FindDeploy(id: Array[Byte])                               extends Command
```

Comment:
DeployId ?

Reply:
`model` doesn't know about `casper`, and that's where `DeployId` is defined

Reply:
ok